py3status/modules/gcal.py

```diff
@@ -0,0 +1,160 @@
+# -*- coding: utf-8 -*-
+"""
+Display upcoming calendar events from Google Calendar.
+
+Configuration parameters:
+    agenda_days: show calendar events for number of days (default 1)
+    button_refresh: mouse button to refresh this module (default 2)
+    cache_timeout: refresh interval for this module (default 3600)
+    command: customize gcalcli to dictate what to do with the events
+        (default 'gcalcli agenda --nocolor')
+    format: display format for this module (default '{format_event}')
+    format_event: display format for events
+        (default '[\?if=is_date&color=#ff0 %a %b %d ]
+        [\?if=is_time&color=#0f0 %-I:%M %p ][\?color=#f00 {event}]')
+    format_separator: show separator only if more than one (default ', ')
+    remove: list of strings to remove from the output (default [])
```

Comment:
@tobes. We could remove this option in favor of global sed-like option.

I added this so the users could `remove = [':00']` turning `7:00 AM` to `7 AM` since it is not possible to do `[\?if=is_time&color=#0f0 %-I[\?has_zero :%M] %p]` due to lack of placeholders. We still keep `%M` for `7:15 AM`, `7:30 AM`, `7:45 AM`, etc.